<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReturnGoodsDialog</class>
 <widget class="QDialog" name="ReturnGoodsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Повернення товару</string>
  </property>
  <widget class="QPushButton" name="return_pb">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Повернути</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pb">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="QTableView" name="sold_goods_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
